tests/src/Simple/PInvoke/PInvoke.cs

```diff
@@ -92,6 +92,21 @@ internal class Program
         [DllImport("*", CallingConvention = CallingConvention.StdCall)]
         static extern bool StructTest_Nested(NestedStruct ns);
 
+        [StructLayout(LayoutKind.Sequential, CharSet= CharSet.Unicode, Pack = 4)]
+        public unsafe struct InlineArrayStruct
+        {
+            public int f0;
+            public int f1;
+            public int f2;
+            public int f3;
+            public int f4;
+            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 128)]
+            public char[] inlineArray;
```

Comment:
Please add a case for strings, especially for ByValAnsiString - the calculation when you don't have enough buffer can be tricky.